src/RoleObjects/SingleLineTextBoxData.js

```diff
@@ -62,7 +62,12 @@ export default class SingleLineTextBoxData extends InputTagData {
    * @param {boolean} enable - true if autocomplete should be enabled, false otherwise
    */
   set autoComplete(enable) {
```

Comment:
Since I think we are trying to keep it backward compatible as well, If I change the function signature, won't this create backward compatibility issue?

Reply:
Changing the signature (specifically the variable names), when it's not a function that takes an object with expected keys, doesn't change the API. The function signature doesn't _technically_ change.

Reply:
We should also mark that it's now boolean or string, right?